Public/index_app/index.js

```diff
@@ -1,7 +1,19 @@
 var indexApp = angular.module('indexApp',[]);
 
-indexApp.controller('greetingController', ['$scope', function($scope){
+indexApp.controller('greetingController', ['$scope', '$http', '$location', function($scope, $http, $location){
+	$scope.dataObject = {};
 	$scope.greeting = "hello";
+	console.log($location);
+	$http.get('http://localhost:4567/files').then(getDataSuccess, getDataError);
```

Comment:
This is a very basic example, tranditionally this would be stored in a service and this $http promise object would be returned in a service function.